sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreChangelog.scala

```diff
@@ -60,6 +60,7 @@ object RecordType extends Enumeration {
     recordType match {
       case PUT_RECORD => "update"
       case DELETE_RECORD => "delete"
+      case MERGE_RECORD => "append"
```

Comment:
Nice change! Now apart from the new PR written for adding a metrics for merge: https://github.com/apache/spark/pull/48317, users should also be able to see count of merge operation of this change feed reader.

Reply:
Yes correct